var/spack/repos/builtin/packages/hpctools/package.py

```diff
@@ -32,7 +32,8 @@ class Hpctools(CMakePackage):
     url      = "ssh://bbpcode.epfl.ch/hpc/HPCTools"
 
     version('develop', git=url)
-    version('3.5.1', tag='3.5.1', git=url, preferred=True)
+    version('3.5.2', tag='v3.5.2', git=url, preferred=True)
```

Comment:
`v` : changed to new tag convention? :)

Reply:
Yes, I guess. I use `v` elsewhere, too.

Reply:
v is being deprecated in some open source projects... anyway, I guess every project always ends up having both :p  
But for the sake of consistency, could we all agree in a convention?

Reply:
I would prefer `v`? :P